app/views/cases/offender_sar/_reason_rejected_step.html.slim

```diff
@@ -0,0 +1,21 @@
+- content_for :heading, flush: true
+  = t('cases.offender_sar.reason_rejected_heading')
+
+= render partial: 'layouts/header'
+
+= GovukElementsErrorsHelper.error_summary @case.object,
+        "#{pluralize(@case.errors.count, t('common.error'))} #{ t('common.summary_error')}", ""
+
+= form_for @case, url: url, as: :offender_sar do |f|
+
+  = f.label :rejected_reason, :class => "form-label-bold"
```

Comment:
The design requires the form-hint span so I created an extra label to meet those requirements, however this has resulted in difficulties in displaying the correct error message within the error summary at the top. Looking at other check box pages they do not have the form-hint, such as https://github.com/ministryofjustice/correspondence_tool_staff/blob/8e1a33e3585d046555815098c55c4fcca171c7f8/app/views/teams/edit.html.slim#L4